### DISCUSSION TITLE
ffmpeg and Lychee 5.5.1

### DISCUSSION THREAD

**Discussion by foxbw** [LycheeOrg/Lychee#2512] [Q&A] [answered]
I have installed Lychee 5.5.1. Now I wanted to install ffmpeg so that my videos can also be provided with thumbnails. I followed the documentation.
However, I have the problem that the VideoEditing.php file does not exist in the specified directory. The file is present on Git, but the file is not present in the zipped download. Other files are also missing.
Does ffmpeg work with version 5.5.1 or should I use another version? 
What can I do to make ffmpef work under 5.5.1?

**Comment by ildyria** [comment]
I do not see VideoEditing in the git could you point me where you see it in the repository. I am not sure what you are talking about.

5.5.1 does support ffmpeg and I use it on some of my small videos.

**Reply by foxbw** [reply]
Okay. On Friday I could see and download the VideoEditing.php. Now there is no file like this. 

So I can't do this from the documentation:
![grafik](https://github.com/user-attachments/assets/c92cfe66-d793-4f5d-a2b4-42029b42f925)

**Reply by ildyria** [reply]
4.2 is quite old.
Now it is in the settings directly, no need to edit files anymore.

**Reply by foxbw** [reply]
I thought these settings are for all versions after 4.2.

Okay, I'll check and test it.

**Reply by ildyria** [reply]
yeah we may have forgotten to update the docs...

**Reply by ildyria** [reply, marked as answer]
![image](https://github.com/user-attachments/assets/2660b271-8836-454f-bf70-50f9a1bb1ec4)
And you will probably also need to set `has_ffmpeg` to `1`

**Reply by foxbw** [reply]
I set the two path details accordingly in Lychee and also set the has_ffmpeg to 1. 
But I still don't see any thumbnails for the videos. What I also noticed is that the video is displayed but cannot be played. Are these two problems related?

Update
I don't know why, but now I can play the videos. Only the thumbnails are not available yet.

**Reply by ildyria** [reply]
> But I still don't see any thumbnails for the videos. What I also noticed is that the video is displayed but cannot be played. Are these two problems related?

Quick question, when you say you don't see thumbnails, is it for newly uploaded videos or for old ones?

**Reply by foxbw** [reply]
You're right. The thumbnails do not work on videos that have already been uploaded. When I upload new videos everything works. Thank you for your help.
